libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/FacebookMessage.cs

```diff
@@ -8,10 +8,21 @@
 namespace Microsoft.Bot.Builder.Adapters.Facebook
 {
     /// <summary>
-    /// Represents the messaging structure.
+    /// Represents the messaging structure. See https://developers.facebook.com/docs/messenger-platform/reference/webhook-events/#messaging.
     /// </summary>
     public class FacebookMessage
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FacebookMessage"/> class.
+        /// </summary>
+        /// <param name="recipientId">Id of the message recipient.</param>
+        /// <param name="message"><see cref="Message"/> representing the contents of the message.</param>
+        /// <param name="messagingType">Property identifying the messaging type of the message being sent. See https://developers.facebook.com/docs/messenger-platform/send-messages#messaging_types. </param>
+        /// <param name="tag">The optional message tag string. See https://developers.facebook.com/docs/messenger-platform/send-messages/message-tags.</param>
+        /// <param name="notificationType">The optional notification type: REGULAR (default value), SILENT_PUSH, NO_PUSH.</param>
+        /// <param name="personalId">The persona Id.</param>
```

Comment:
'personalId' shouldn't be called 'personaId'?

Reply:
yeap